### ISSUE TITLE
SQL parser issue - 2.1 - unable to create embedded list item

### ISSUE THREAD

**Issue by seeden** [orientechnologies/orientdb#4804]
Hi

I have a problem to create any embedded list item because parser is not able to parse *@type* and @class fields.

orientdb version: 2.1

Here is an output from orientdb:
```sh
OrientDB.RequestError: Error on parsing command at position #0: 
Encountered " <RECORD_ATTRIBUTE> "@type "" at line 1, column 400.
Was expecting one of:
    <TO> ...
    <VALUES> ...
    <SET> ...
    <ADD> ...
    <PUT> ...
    <MERGE> ...
    <CONTENT> ...
    <REMOVE> ...
    <ORDER> ...
    <GROUP> ...
    <OFFSET> ...
    <RECORD> ...
    <LUCENE> ...
    <NEAR> ...
    <WITHIN> ...
    "`" ...
    <IN> ...
    <KEY> ...
    <IDENTIFIER> ...
```

**Comment by seeden** [comment]
Maybe it is linked with https://github.com/orientechnologies/orientdb/issues/4724

**Comment by luigidellaquila** [comment]
Hi @seeden 

could you please post the SQL statement you are using?

Thanks

**Comment by seeden** [comment]
I have a classes with defined schemas. This is the reason why I need to add metadata @type and @class if I wanted to create an embedded item. Without it I am not able to add any embedded item and migrate to version 2.1. Thank you for your help Luigi.

Here is a query:

```sql
INSERT INTO Activity SET user = #14:1, story = #18:2, like = :paramlike0;

{ 
  params: {
    paramlike0: {
      count: 0, 
      latest: [], 
      '@type': 'document', 
      '@class': 'Like'
    }
  }
}
```

**Comment by seeden** [comment]
Any progress with this issue?

**Comment by luigidellaquila** [comment]
fixing it right now

**Comment by seeden** [comment]
Thank you

**Comment by luigidellaquila** [comment]
fixed in both 2.1.x and develop branches.
Please confirm that it also fixes your use case

Thanks

**Comment by seeden** [comment]
Thanks it is working
